ports/stm32/usbd_cdc_interface.c

```diff
@@ -267,7 +266,7 @@ int8_t usbd_cdc_receive(usbd_cdc_state_t *cdc_in, size_t len) {
 
     // copy the incoming data into the circular buffer
     for (const uint8_t *src = cdc->rx_packet_buf, *top = cdc->rx_packet_buf + len; src < top; ++src) {
-        if (cdc->attached_to_repl && mp_interrupt_char != -1 && *src == mp_interrupt_char) {
+        if (cdc->attached_to_repl && MP_STATE_VM(interrupt_char) != -1 && *src == MP_STATE_VM(interrupt_char)) {
```

Comment:
Should this be `MP_INTERRUPT_CHAR_DISABLE_AND_CANCEL_PENDING` instead of `-1`?

Reply:
Hmm, it should actually test for `<0` because now `interrupt_char` could be -1 or -2... maybe it could do with a macro like `MP_INTERRUPT_CHAR_ENABLED()`.

Reply:
Now fixed.  I just removed this part of the if-logic, it's not needed.